lib/index.js

```diff
@@ -58,7 +62,7 @@ Chartbeat.prototype.page = function(page) {
     this._ready = false;  // switch ready to false so that no pages after the first one can fire until _initialize has loaded chartbeat script
     this.pageCalledYet = true;
     this._initialize();
-  } else {
+  } else if (this.options.trackGalleryClicksAsPageviews || urlChanged) {
```

Comment:
It feels like the name of this option is very specific for user's that have image galleries. Are there potentially other types of sites that might use this feature?